packages/types/src/organizationMembership.ts

```diff
@@ -8,6 +8,11 @@ interface Base {
   role: string;
 }
 
+interface Placeholder {
+  permission: unknown;
+  role: unknown;
+}
+
```

Comment:
❓ isn't this change added in https://github.com/clerk/javascript/pull/2314 ?

Reply:
It is, it was the only way I could get this to build and test, as it wasn't available at the time. My plan was to rebase everything and weed it out once it came time.